OpenRA.Game/Graphics/WorldRenderer.cs

```diff
@@ -133,7 +133,27 @@ void GenerateRenderables()
 			foreach (var e in World.ScreenMap.RenderableEffectsInBox(Viewport.TopLeft, Viewport.BottomRight))
 				renderablesBuffer.AddRange(e.Render(this));
 
-			renderablesBuffer.Sort((x, y) => RenderableZPositionComparisonKey(x).CompareTo(RenderableZPositionComparisonKey(y)));
+			renderablesBuffer.Sort((x, y) =>
```

Comment:
Can we rename this to a, b? Imo it is a bit confusing to have the variables named x and y and then do comparisons with x and y positions.

Reply:
Done.